corehq/apps/userreports/fixtures.py

```diff
@@ -65,6 +69,8 @@ def __call__(self, user, version, last_sync=None):
                 reports_elem.append(self._report_config_to_fixture(report_config, user))
             except UserReportsError:
                 pass
+            except Exception as err:
+                logger.exception('Error generating report fixture: {}'.format(err))
```

Comment:
I don't think I really thought about it. I just created a new logger out of habit. 

I'm a fan of consistent. I'll use `logging.exception` instead. And I'll go look up why & when to create new loggers instead of falling back on old habits. :suspect: